samples/highcharts/sonification/chart-earcon/demo.js

```diff
@@ -1,3 +1,11 @@
+Highcharts.setOptions({
+    lang: {
+        accessibility: {
+            chartContainerLabel: 'Click play button to sonify chart.'
```

Comment:
The default label is the chart title + "Highcharts interactive chart", and I think that works fine here - unless you disagree?